support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.jsx

```diff
@@ -213,7 +213,7 @@ function setUpPaymentListener(props: PropTypes, paymentRequest: Object, paymentM
 
     const stateUpdateOk =
       props.stripeAccount !== 'ONE_OFF' && (props.countryGroupId === UnitedStates || props.countryGroupId === Canada) ?
-      updatePayerState(token, props.updateState) : true;
+        updatePayerState(token, props.updateState) : true;
```

Comment:
Are these just linting changes? John Duffell would definitely ask you to remove these 😉